### PR TITLE
Add verification function when saving

### PR DESCRIPTION
**Pull request by abliger**

### CODE REVIEW

**Review by abliger — COMMENTED** [review]
#281

File: package.json [hunk 1]
```diff
@@ -59,6 +59,12 @@
           "scope": "resource",
           "description": "List of local or remote style sheets for suggestions.",
           "default": []
+        },
+        "vscode-html-css.vaildOnSave": {
+          "type": "boolean",
+          "description": "Verify label class names when saving files.",
+          "scope": "application",
```
Comment (ecmel) [inline package.json:66]: Better to be resource (workspace) scopped and be an enum for future. (Like On save, On change etc)

File: src/extension.ts [hunk 2]
```diff
@@ -27,8 +27,13 @@ export function activate(context: ExtensionContext) {
     workspace.onDidCloseTextDocument((document) =>
       validations.delete(document.uri)
     ),
-    workspace.onDidChangeTextDocument((event) =>
-      validations.delete(event.document.uri)
+    workspace.onDidChangeTextDocument(async (event) => {
```
Comment (ecmel) [inline src/extension.ts:30]: This should be according to the settings, such as on save on change etc.

File: src/settings.ts [hunk 3]
```diff
@@ -16,3 +16,9 @@ export function getStyleSheets(uri: Uri): string[] {
     .getConfiguration("css", uri)
     .get<string[]>("styleSheets", []);
 }
+
+export function getVaildOnSave(): Boolean {
+  return workspace
```
Comment (ecmel) [inline src/settings.ts:21]: Should be resource scopped like styleSheets

**Comment by ecmel** [comment]
Thank you for the PR 👍

File: src/extension.ts (hunk 2 above)
Comment (abliger) [reply src/extension.ts:30]: I also discovered this issue today and will submit it later

File: src/settings.ts (hunk 3 above)
Comment (abliger) [reply src/settings.ts:21]: Is vscode-html-css modified to css?

**Review by abliger — COMMENTED** [review]
Add feature: Prompt source file

**Pull request merged**